### ISSUE TITLE
dump() does not throw exception as shown in the docs

### ISSUE THREAD

**Issue by samayo** [symfony/symfony-docs#7516]
There seems to be a mistake in the doc, right [here](https://github.com/symfony/symfony-docs/blob/3.2/components/yaml.rst#handling-invalid-types). It shows the code in the second example throws an error, which I will copy here as: 

```
$data = new \stdClass(); // by default objects are invalid.
Yaml::dump($data, Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE); // throws an exception

echo $yaml; // { foo: bar }
```

I have Yaml `3.2.3` and I can't produce the same result, also the third line doesn't seem to have any effect than throw a syntax error, since it is undefined. Even if it was defined in the previous block, the output isn't as shown in the example above.

**Comment by xabbuh** [comment]
That's indeed an issue. The example is missing the `$inline` and `$indent` arguments (see #7518 for the fix).

**Comment by javiereguiluz** [comment]
Closing it as fixed by #7518.
